SendiSteve/BootcampChallengeTwo
# Created unit tests for the route pages

#### What's the PR do?
Its an implementation of the unit tests for the route pages that allow the user access a specific url in the browser

#### Where should the reviewer start?
Changes made in the template folder and the app.py file. Have added 5 files ie the index,welcome,recipe, login and signup pages

#### How should this be manually tested?
Ensure that pytest is installed 
In the your terminal type the command pytest tests.py and you will be able to see failed and passed tests

#### Any background context you want to provide?
I am still working on more tests
